src/opencl_mask_extras.h

```diff
@@ -14,8 +14,6 @@
 #ifndef _JOHN_OPENCL_MASK_H
 #define _JOHN_OPENCL_MASK_H
```

Comment:
I see a bug here: this should be `_JOHN_OPENCL_MASK_EXTRAS_H` or something, and definitely not same guard as in opencl_mask.h